asv/runner.py

```diff
@@ -421,12 +421,8 @@ def get_spawner(env, benchmark_dir, launch_method):
     has_fork = hasattr(os, 'fork') and hasattr(socket, 'AF_UNIX')
 
     if launch_method in (None, 'auto'):
-        # Don't use ForkServer as default on OSX, because many Apple
-        # things are not fork-safe
-        if has_fork and sys.platform not in ('darwin',):
-            launch_method = "forkserver"
-        else:
-            launch_method = "spawn"
+        # Don't use ForkServer as default as threading and fork are not compatible
+        launch_method = "spawn"
```

Comment:
Do we keep `spawn` as the default in all cases, as suggested by the `fork()` documentation, although it's slower?

Reply:
I think we should. The time of `spawn` should only be important when running tests, in "real" usage the benchmark run time should be much more than the difference between "spawn" and "fork"